### PR TITLE
Add styles for small checkbox cards

### PR DESCRIPTION
**Pull request by aknobloch3**
- Added a class `.small` that can be added to radio cards and checkbox cards.
- Added `$card-checkbox--padding` so you can overwrite the default padding if you want

### CODE REVIEW

**Comment by aknobloch3** [comment]
screen shot 2018-01-29 at 3 44 42 pm

**Comment by aknobloch3** [comment]
screen shot 2018-01-29 at 3 45 25 pm

File: example/templates/checkboxes/example.html [hunk 1]
```diff
@@ -217,6 +218,21 @@ <h2 class="example-header no-margin--top">Checkbox Cards</h2>
 					<label for="checkbox-test3" class="card disabled">Disabled</label>
 				</div>
 			</fieldset>
+			<fieldset class="column column--full">
+				<label>Small Checkbox Card</label>
+				<div class="checkbox-card small">
```
Comment (jrauh) [inline example/templates/checkboxes/example.html]: Questions around naming. For small buttons, we use the class `.button--small`. Would `.checkbox-card--small` make more sense?

Comment (aknobloch3) [reply example/templates/checkboxes/example.html]: Yeah, good point. I know for spinner we do `dash-spinner small`. What is better? I feel like it is a difference between being descriptive and too long of class names (and repetitive)...

Comment (jrauh) [reply example/templates/checkboxes/example.html]: https://github.com/dashframework/dashing/tree/master/guidelines/naming-conventions

Per our current guidelines, I think `.checkbox-card--small` is best. It does make naming more wordy, but I think specificity here is good. It tells you exactly what the class does. Just adding `.small` might not provide that clarity. Is it small text? A small card? Etc.

**Comment by aknobloch3** [comment]
Merged
`v1.2.3`

**Pull request closed without merging**